README.md

```diff
@@ -1,3 +1,94 @@
-# geomancer
+![Geomancer logo](https://i.imgur.com/Zj5Gcd6.jpg)
 
-Automated OSM Feature Engineering built on top of BigQuery
+---
+
+Geomancer is a geospatial feature engineering Python package for OpenStreetMap (OSM) data that are saved in BigQuery. This project is currently in pre-alpha and is under active development.
+
+
+# Dependencies
+
+- Python 3.6.0+
```

Comment:
Just remove this list na lang. I realized that we'd have different dependencies for different backends. So if you want `geomancer[bq]`, you'd depend on `google-cloud-bigquery`, etc. But if you only want `geomancer[spatialite]`, then you probably won't need `google-cloud-bigquery`.

Reply:
I'll remove the dependencies header and bullet points.